src/sio.spec.ts

```diff
@@ -1,11 +1,10 @@
 import { RP2040, SIO_START_ADDRESS } from './rp2040';
-
 import { GDBClient } from './utils/gdbclient';
 import { ICortexTestDriver } from './utils/test-driver';
 import { GDBTestDriver } from './utils/test-driver-gdb';
 import { RP2040TestDriver } from './utils/test-driver-rp2040';
 
-//HARDWARE DIVIDER
+//HARDWARE DIVIDER REGISTERS ABSOLUTE ADDRESS
```

Comment:
Can you please change to 

`// Hardware Divider registers absolute address`

(so not all letters are capital)